pkg/controller/core/workload_controller.go

```diff
@@ -139,32 +139,32 @@ func (r *WorkloadReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 	case pending:
 		if !r.queues.QueueForWorkloadExists(&wl) {
 			err := workload.UpdateStatusIfChanged(ctx, r.client, &wl, kueue.WorkloadAdmitted, metav1.ConditionFalse,
-				"Inadmissible", fmt.Sprintf("LocalQueue %s doesn't exist", wl.Spec.QueueName))
+				"Inadmissible", fmt.Sprintf("LocalQueue %s doesn't exist", wl.Spec.QueueName), constants.AdmissionName)
```

Comment:
NB: This should have been `kueue-workload-controller`, but we will be merging the Admitted condition along with the changes for `.status.admission` #603